pkg/kubeapiserver/restmanager.go

```diff
@@ -291,7 +291,7 @@ func (m *RESTManager) genLegacyResourceRESTStorage(gvr schema.GroupVersionResour
 }
 
 func (m *RESTManager) genCustomResourceRESTStorage(gvr schema.GroupVersionResource, kind string) (*resourcerest.RESTStorage, error) {
-	storageConfig, err := m.resourcetSorageConfig.NewCustomResourceConfig(gvr)
+	storageConfig, err := m.resourcetSorageConfig.NewCustomResourceConfig(gvr, false)
```

Comment:
The caller of the `genCustomResourceRESTStorage` and `genLegacyResourceRESTStorage` knows if the resource is namespaced.
https://github.com/clusterpedia-io/clusterpedia/blob/af5b4a2db0655be68f16e9dfc30a90fdbbe02946/pkg/kubeapiserver/restmanager.go#L227-L230

```go
 	storage, err = m.genLegacyResourceRESTStorage(gvr, info.APIResource.Kind, infor.APIResource.Namespaced) 
```

Reply:
> The caller of the `genCustomResourceRESTStorage`/`genLegacyResourceRESTStorage` know if the resource is namespaced.
> 
> https://github.com/clusterpedia-io/clusterpedia/blob/af5b4a2db0655be68f16e9dfc30a90fdbbe02946/pkg/kubeapiserver/restmanager.go#L227-L230
> 
> ```go
>  	storage, err = m.genLegacyResourceRESTStorage(gvr, info.APIResource.Kind, infor.APIResource.Namespaced) 
> ```

thx, it's been fixed, please approve running workflows again~